src/common/home/Home.jsx

```diff
@@ -31,6 +36,32 @@ const Home = () => {
     });
   }, [data, setSearchTerm, searchTerm, setFilterQuery]);
 
+  //* array of tweet IDs to show on the home page
+    const tweetIdArray = [
+        "1544376341172068352",  
+        "1530197614771458049",
+        "1529884210269671424",
+        "1522165831005728769",
+        "1521784126717710336",
+        "1518200201302974464",
+        "1541802575775035392",
+        "1515598358748237830",
+        "1528938773710782464",
+        "1532349503709122561"
+    ]
+
+    //* set the state for loading
+   
+    const [isLoading,setLoading] = useState(true)
```

Comment:
Rename the variables as isTweetsLoading and setTweetsLoading

Reply:
ok